src/devices/Lp55231/RgbLed.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device.Lp55231
+{
+    /// <summary>
+    /// Represents an RgbLed on the Lp55231
+    /// </summary>
+    public class RgbLed
+    {
+        private readonly Action<byte, byte> _setIntensity;
+        private readonly byte _instance;
+
+        private byte _red;
+        private byte _green;
+        private byte _blue;
+
+        internal RgbLed(byte instance, Action<byte, byte> setIntensity)
+        {
+            if (instance < 0 || instance > 2)
+            {
+                throw new IndexOutOfRangeException("Channel must be between 0 and 2");
+            }
+
+            if (setIntensity == null)
+            {
+                throw new ArgumentNullException(nameof(setIntensity));
+            }
+
+            _instance = instance;
+            _setIntensity = setIntensity;
+        }
+
```

Comment:
Maybe we can add a Color property directly?

Reply:
I initially considered using Color but decided against it as:
1) It meant taking a dependency on another assembly (System.Drawing); and
2) It didn't allow color components to be changed independently.

Despite this, given your suggestion, I have refactored to use Color. I will add the ability to change individual channel intensities at a later date.